tests/integration/questionnaire/test_questionnaire_locale.py

```diff
@@ -0,0 +1,21 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestSession(IntegrationTestCase):
+    def test_none_language_code_defaults_to_en_GB(self):
```

Comment:
Minor, but we now have 1 test via Integration test and another in functional test. Can we not just update the existing JS test to cover this scenario?.

Reply:
We need code coverage, so opted for int test